pkg/controller/operconfig/operconfig_controller.go

```diff
@@ -51,18 +58,72 @@ var ManifestPath = "./bindata"
 // Add creates a new OperConfig Controller and adds it to the Manager. The Manager will set fields on the Controller
 // and Start it when the Manager is Started.
 func Add(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.Client) error {
-	return add(mgr, newReconciler(mgr, status, c))
+	rc, err := newReconciler(mgr, status, c)
+	if err != nil {
+		return err
+	}
+	return add(mgr, rc)
 }
 
 const ControllerName = "operconfig"
 
 // newReconciler returns a new reconcile.Reconciler
-func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.Client) *ReconcileOperConfig {
-	return &ReconcileOperConfig{
-		client: c,
-		status: status,
-		mapper: mgr.GetRESTMapper(),
+func newReconciler(mgr manager.Manager, status *statusmanager.StatusManager, c cnoclient.Client) (*ReconcileOperConfig, error) {
+	ctx := context.Background()
+	ctx, cancelFn := context.WithCancel(ctx)
+	go func() {
+		defer cancelFn()
+	}()
```

Comment:
I would just have these bits close to  `go featureGateAccessor.Run(ctx)`
And we don't actually need the go routine?

Reply:
changing this to `go featureGateAccessor.Run(context.TODO())`